pkgs/development/python-modules/reedsolo/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, cython, nose }:
+
+buildPythonPackage rec {
+  pname = "reedsolo";
+  version = "1.5.4";
+
+  # Pypi does not have the tests
+  src = fetchFromGitHub {
+    owner = "tomerfiliba";
+    repo = "reedsolomon";
+    # https://github.com/tomerfiliba/reedsolomon/issues/28
+    rev = "73926cdf81b39009bd6e46c8d49f3bbc0eaad4e4";
+    sha256 = "03wrr0c32dsl7h9k794b8fwnyzklvmxgriy49mjvvd3val829cc1";
+  };
+
+  buildInputs = [ cython ];
```

needs to be executable for cross platform
```suggestion
  nativeBuildInputs = [ cython ];
```